_scripts/install

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+BIN=$(dirname ${BASH_SOURCE})
+source ${BIN}/funcs.sh
+set -e
+
+check_var ENV_FILE ROOT_ENV ROOT_DIR
+
+${BIN}/confirm yes "This tool is provided for your convenience, please read the README of each project before installation."
```

Comment:
Without providing a `default` argument, `_scripts/confirm` adds a period after your question; so having a period after your question results in 2 periods. 

Remove the period:
`${BIN}/confirm yes "This tool is provided for your convenience, please read the README of each project before installation"`